### PR TITLE
Update CMEK FAQ

### PR DESCRIPTION
**Pull request by mdlinville**
Update CMEK Faq
- Request: https://cockroachlabs.slack.com/archives/C02Q8MLFFE0/p1677380623240759
- Add two questions
- Update one question
- Reorganization

## Previews
[cockroachcloud/cmek-faq.md](https://deploy-preview-16365--cockroachdb-docs.netlify.app/docs/cockroachcloud/cmek-faq.html)

## Tests
- [x] Local build
- [x] Linkcheck

### CODE REVIEW

File: cockroachcloud/cmek-faq.md [hunk 1]
```diff
@@ -13,6 +13,22 @@ This article provides answers to common questions about [Customer-Managed Encryp
...
+
+## Can we enable CMEK for an existing cluster that wasn't created as a private cluster?
+
+CMEK can be enabled only on a private cluster. Contact your account team for advice about how to migrate or restore your existing cluster's data to a new private cluster.
+
+## If we enable CMEK for a cluster that has been in use for some time, is the existing data encrypted at that time?
+
+{{ site.data.products.dedicated }} does not force encryption of previously-written data but instead relies on normal storage engine churn for desired encryption. That means the new key is used to encrypt newly-written data, while previously-written data remains unencrypted unless it’s rewritten.
+
+## Can we enable CMEK when we add a new region to a cluster?
```
Comment (abhinavg6) [inline cockroachcloud/cmek-faq.md:28]:
```suggestion
## Can we enable CMEK for a new region when it's added to a CMEK-enabled cluster?
```

File: cockroachcloud/cmek-faq.md [hunk 2]
```diff
@@ -13,6 +13,22 @@ This article provides answers to common questions about [Customer-Managed Encryp
...
+
+Yes, when you add a new CMEK-enabled region to a cluster, you must also enable CMEK on its existing regions if it is not already enabled. Refer to [Activate CMEK for your {{ site.data.products.dedicated }} Cluster](/docs/cockroachcloud/managing-cmek.html#step-4-activate-cmek-for-your-cockroachdb-dedicated-cluster) and [Add a Region to a CMEK-enabled Cluster](/docs/cockroachcloud/managing-cmek.html#add-a-region-to-a-cmek-enabled-cluster).
```
Comment (abhinavg6) [inline cockroachcloud/cmek-faq.md:30]:
```suggestion
Yes, when you add a new region to a CMEK-enabled cluster, you can enable CMEK for that region. Refer to [Add a Region to a CMEK-enabled Cluster](/docs/cockroachcloud/managing-cmek.html#add-a-region-to-a-cmek-enabled-cluster).
```

**Review by abhinavg6 — CHANGES_REQUESTED** [review]
Requested change to context of one of the questions.

File: cockroachcloud/cmek-faq.md [hunk 3]
```diff
@@ -13,6 +13,22 @@ This article provides answers to common questions about [Customer-Managed Encryp
...
+
+Contact your Cockroach Labs account team to enroll your organization in limited access for CMEK. CMEK can only be enabled on a private cluster, so your organization must also be enrolled in limited access for private clusters.
+
+## Can we enable CMEK for an existing cluster that wasn't created as a private cluster?
+
+CMEK can be enabled only on a private cluster. Contact your account team for advice about how to migrate or restore your existing cluster's data to a new private cluster.
+
+## If we enable CMEK for a cluster that has been in use for some time, is the existing data encrypted at that time?
+
+{{ site.data.products.dedicated }} does not force encryption of previously-written data but instead relies on normal storage engine churn for desired encryption. That means the new key is used to encrypt newly-written data, while previously-written data remains unencrypted unless it’s rewritten.
```
Comment (taroface) [inline cockroachcloud/cmek-faq.md:26]:
```suggestion
{{ site.data.products.dedicated }} does not force encryption of previously-written data but instead relies on normal storage engine churn for desired encryption. That means the new key is used to encrypt newly-written data, while previously-written data remains unencrypted unless it's rewritten.
```

**Review by taroface — APPROVED** [review]
LGTM. Reformatted an apostrophe.

**Review by abhinavg6 — APPROVED** [review]

**Pull request merged**